ubiconfig/_impl/loaders/local.py

```diff
@@ -44,6 +44,23 @@ def load_all(self, recursive=False):
 
         return ubi_configs
 
+    def load_by_cs_label(self, cs_label):
+        ubi_configs = self.load_all(recursive=True)
+
+        for config in ubi_configs:
+            for cs in [
+                config.content_sets.rpm.input,
+                config.content_sets.rpm.output,
+                config.content_sets.srpm.input,
+                config.content_sets.srpm.output,
+                config.content_sets.debuginfo.input,
+                config.content_sets.debuginfo.output,
+            ]:
+                if cs == cs_label:
+                    return config
+
+        raise ConfigNotFound("No config found for label: %s" % cs_label)
+
```

Comment:
As Rohan pointed out in the previous comment, you should implement the new API in the base class, so you can avoid this duplication.